src/components/macro/macro.component.scss

```diff
@@ -59,18 +59,22 @@ h1 {
 
 .macro-actions-container {
     overflow: auto;
-    max-height: 70vh;
+    max-height: 78vh;
     margin-bottom: 0;
     border-radius: 4px 4px 0 0;
     border: 1px solid #ddd;
     border-bottom: 0;
 
-    @media screen and (min-height: 1000px) {
+     @media screen and (min-height: 730px) {
```

Comment:
These are not working for me, otherwise I think it should be done with flexbox, instead of media query.